core/src/utils/rtl/dir.ts

```diff
@@ -0,0 +1,14 @@
+
+/**
+ * Returns `true` if the document or host element
+ * has a `dir` set to `rtl`. The host value will always
+ * take priority over the root document value.
+ */
+export const isRTL = (hostEl?: Pick<HTMLElement, 'dir'>) => {
+  if (hostEl) {
+    if (hostEl.dir !== '') {
+      return hostEl.dir.toLowerCase() === 'rtl';
```

Comment:
Should we be converting to lowercase? It looks `"Ltr"` should be treated as invalid looking at the accepted values: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir

Reply:
I'm happy to swap that to explicit checks. Was siding on Dx, but if the browser will ignore any values that aren't `rtl | ltr | auto`, that isn't very helpful to be different.

Reply:
Oops I take it back. The browser `dir` is not case sensitive, so what you have currently aligns with how the browser behaves: https://codepen.io/liamdebeasi/pen/Jjrvgrp

Reply:
Haha, that change is reverted 👍